include/openssl/ssl3.h

```diff
@@ -25,6 +25,9 @@
 extern "C" {
 #endif
 
+/* Flag used on OpenSSL ciphersuite ids to indicate they are for SSLv3+ */
+# define SSL3_CK_CIPHERSUITE_FLAG                0x03000000
```

Comment:
Can't. The SSLv3 flag is already set on every ciphersuite id - which are already in the public header files - so it would be a binary compatibility issue to change it. The only thing I'm doing here is creating a separate define just for the flag itself to avoid the use of a magic number at one point in the code. It is otherwise quite incidental to this change.

However, looking at this again, I think this define should probably be in ssl_locl.h rather than in the public header - so I moved it there.